pallets/asset/src/lib.rs

```diff
@@ -503,12 +505,12 @@ decl_module! {
         ///
         /// ## Permissions
         /// * Portfolio
-        #[weight = <T as Trait>::WeightInfo::create_asset(
+        #[weight = <T as Trait>::WeightInfo::create_asset_and_mint(
             name.len() as u32,
             identifiers.len() as u32,
             funding_round.as_ref().map_or(0, |name| name.len()) as u32
         )]
-        pub fn create_asset(
+        pub fn create_asset_and_mint(
```

Comment:
This is not needed, let's remove it. Users can batch call `create_asset` and `issue` to get this functionality.